runtime/themes/nyxvamp-radiance.toml

```diff
@@ -0,0 +1,121 @@
+# nyxvamp - radiance variant
+# author: <zoey.spessanha@zeetech.io>
+
+# UI Elements
```

Comment:
It looks like there isn't theming yet for `ui.virtual` keys. I would recommend setting `ui.virtual` to a color close to what's used for comments and also `ui.virtual.ruler = { bg = "<some comment-like color>" }` (specifically setting a background for this one)

For rulers in particular you can test with `:set rulers [80]`